src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -1291,7 +1291,7 @@ public String getValue() {
 
             @Override
             public String getDisplayName() {
-                return null;
+                return "";
```

Comment:
This seems to be not related to the parent pom upgrade. What's different on returning `null` or empty string? Is it fixing a bug?